web/src/stories/ShowAllModal.stories.tsx

```diff
@@ -0,0 +1,291 @@
+import React from "react";
+import { ComponentMeta } from "@storybook/react";
+
+import ShowAllModal from "@src/components/modals/ShowAllModal";
+import dayjs from "dayjs";
+
+export default {
+  title: "Example/ShowAllModal",
+  component: ShowAllModal,
+} as ComponentMeta<typeof ShowAllModal>;
+
+export const Volumes = () => {
+  const renderShowAllVolumes = () => {
+    const volumes = [
+      {
```

Comment:
maybe there's a better place to store fake data?

Reply:
Looks like this is how BPR does it: https://github.com/alan2207/bulletproof-react/blob/master/src/test/data-generators.ts

something like this could be neat for using with react-query too since it just takes a promise returning function for fetch which doesn't necessarily have to do a network call.

Reply:
great find! I'll start doing this for subsequent PRs.